android-core/src/main/java/com/mparticle/identity/MParticleUser.java

```diff
@@ -129,4 +131,10 @@ public interface MParticleUser {
      */
     long getLastSeenTime();
 
+    /**
+     * query the user audience
```

Comment:
```suggestion
     * Get a list of the audiences that this given MPID is currently within.
```

@Mansi-mParticle please also consult the iOS code to align on the javadocs.